config/locales/en.yml

```diff
@@ -405,6 +405,10 @@ en:
       email_notifications_question: 'How can I modify email notifications?'
       email_notifications_answer: 'Go to the <strong>%{accounts}</strong> page. You can modify your email, password, profile picture, and bio here. You can also delete your account. Once you delete your account, all your data is deleted.'
     home:
+      update_pwd_modal:
+        title: 'Update password'
+        body: 'We have enforced better security measures for our users, please %{update_password_link}'
+        update_link: 'update your password.'
       stories: Stories
```

Comment:
@julianguyen, is the text ok, as this will be shown for every six months.

Reply:
Hmmmm I think there's also security issues with changing one's password periodically. I think it would be better to just ask the a user who hasn't had a history at all of changing their password to do so.

Reply:
Let's increase this to 12 months!